CONTRIBUTING.md

```diff
@@ -81,11 +81,20 @@ ansible-test units --docker -v --python 3.8 tests/unit/plugins/modules/net_tools
 
 The following commands show how to run integration tests:
 
+#### In docker
+
 ```.bash
-# Run integration tests for the interfaces_files module in a Docker container using the
-# fedora35 operating system image (the supported images depend on your ansible-core version):
-ansible-test integration --docker fedora35 -v interfaces_file
+# Run integration tests in a Docker container for a specific module, e.g. community.general.pacman:
+ansible-test integration -v --docker default pacman
+
+# Integration test on a specific docker image. Find out which target docker images are supported by
+# running ansible-test integration --help and choose one, e.g. fedora35:
+ansible-test integration -v --docker fedora35
```

Comment:
I would update this to `fedora40`, since `fedora35` is already pretty old.

Reply:
The reason I left is as-is, is because no matter the value we update this to, it will get out-of-date. My reasoning is that specifying an older version might make contributors more likely to halt before copy-pasting the command. I was even debating setting it to ubuntu1004 just to drive home that the command is not copy-pastable.

If you do not agree, I can change it.

Reply:
I would bump it to the latest when we change the text anyway, to reduce the amount of friction for folks who simply copy'n'paste.